code-challenges.test.js

```diff
@@ -17,78 +17,173 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var num1 = 15
-// Expected output: "15 is divisible by three"
-
-var num2 = 0
-// Expected output: "0 is divisible by three"
-
-var num3 = -7
-// Expected output: "-7 is not divisble by three"
-
-
-
-
+// Name of the function will be 'divisibleByThree'
+describe("divisbleByThree", () => {
+    var num1 = 15
```

Comment:
awesome test assertions! Usually if the variable is only used within the `it()` block, it should be declared within the it block.